apps/mobile/src/navigation/RootStackNavigator.tsx

```diff
@@ -53,7 +60,17 @@ export function RootStackNavigator({ navigationContainerRef }: Props) {
       initialRouteName={isLoggedIn ? 'MainTabs' : 'Login'}
     >
       <Stack.Screen name="Login" component={LoginStackNavigator} />
-      <Stack.Screen name="MainTabs" component={MainTabNavigator} />
+
+      <Stack.Screen name="Post" component={ProfilePicturePickerScreen} />
+      <Stack.Screen
+        name="ProfilePicturePickerContract"
+        component={ProfilePicturePickerContractScreen}
```

Comment:
question since I'm less familiar with the RootStackNavigator - how come ProfilePicturePickerContractScreen was moved from MainTabStackNavigator to here?

Reply:
The `ProfilePicturePickerContractScreen` was not moved from `MainTabStackNavigator` to `RootTabStackNavigator` but I was adding a new one. 

The reason is in post user flow, we want it to fullscreen (without any tab bar)